### PR TITLE
Switchfix for themeswitcher into develop

### PR DESCRIPTION
**Pull request by itggot-jonathan-lindqvist**

### CODE REVIEW

**Review by bijanfatha — APPROVED** [review]
NOICE

File: film/src/App.vue [hunk 1]
```diff
@@ -46,6 +51,49 @@ export default Vue.extend({
       //this.loading = true
       console.log("WHAT THE FUCK!!!")
       //this.loading = false
+    },
+    changeTheme: function(){
+      // console.log("HSDAKSLDHL")
```
Comment (simon-ulrik) [inline film/src/App.vue:56]: remove

File: film/src/App.vue [hunk 2]
```diff
@@ -46,6 +51,49 @@ export default Vue.extend({
...
+      if(this.theme == false){
+        this.headerColor = "black"
+        this.appColor = "Black"
+        this.searchTitleColor = "#8d2663"
+        this.theme = true
+        // console.log("cancer")
```
Comment (simon-ulrik) [inline film/src/App.vue:62]: remove

File: film/src/App.vue [hunk 3]
```diff
@@ -46,6 +51,49 @@ export default Vue.extend({
...
+        }else{
+          this.$children[3].$data.movieColor = "#8d2663"
+          this.$children[3].$data.plotColor = "white"
+        }
+      }else{
+        //here
+        this.headerColor = "DodgerBlue"
+        this.appColor = "white"
+        this.theme = false
+        // console.log("cancer")
```
Comment (simon-ulrik) [inline film/src/App.vue:81]: remove

File: film/src/App.vue [hunk 4]
```diff
@@ -86,6 +134,24 @@ export default Vue.extend({
     min-height: 100px
     max-height: 100px
 
+  // .app-dark
```
Comment (simon-ulrik) [inline film/src/App.vue:137]: remove

**Review by simon-ulrik — APPROVED** [review]

**Pull request merged**